packages/acceptit-Tests.package/ACScenarioWithSpaceTest.class/instance/testItAllowsScenariosWithSpacesInTItle.st

```diff
@@ -0,0 +1,8 @@
+as yet unclassified
+testItAllowsScenariosWithSpacesInTItle
+
+	self assert: 'm vus' equals: scenarioMethod getSource asString lines first.
```

Comment:
I will write a helper method for this

Reply:
hm I am unable to switch to your branch 
![image](https://user-images.githubusercontent.com/29160369/41690153-78636336-74f4-11e8-8178-0c1d1e47fd20.png)

Reply:
Must be some sort of git/squot error. Not sure where that comes from.

Reply:
Yeah me neither, now I am unable to commit, switch branches and push ... winning...not